test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -155,7 +169,7 @@ test
     const pageTitle = challengePhonePageObject.getFormTitle();
     const pageSubtitle = challengePhonePageObject.getFormSubtitle();
     const primaryButtonText = challengePhonePageObject.getSaveButtonLabel();
-    const secondaryButtonText = challengePhonePageObject.getSecondaryLinkText();
+    const secondaryButtonText = challengePhonePageObject.getSecondaryLinkText('Receive a voice call instead');
```

Comment:
Instead of getting the text using the text and then asserting on it, I think it would be simpler to just assert the button with this text exists. 

You can use this function from the BaseFormObject: `getButton` and just pass in the button label like: `getButton('Receive a voice call instead');` 

So ultimately, it would be like this: 

```
await t.expect(
   await challengePhonePageObject.form.getButton('Receive a voice call instead').exists
).eql(true);
```

Then you can get rid of the `getSecondaryLinkText` since this should work for both v2 and v3.

Reply:
<img width="400" alt="Screen Shot 2023-01-25 at 2 21 04 PM" src="https://user-images.githubusercontent.com/27521424/214705061-573d585c-8770-4370-a328-367cce947e85.png">
Not working in v2, since the secondary text is NOT a button in v2. Might have to stick to the old way

Reply:
Ahh, that stinks, ok, no worries.